<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfiguracion">
    <property name="title">
     <string>Configuracion</string>
    </property>
    <addaction name="actionAlmacenamiento"/>
    <addaction name="actionComunicacion"/>
    <addaction name="actionGetCantidad"/>
   </widget>
   <widget class="QMenu" name="menuAcceso">
    <property name="title">
     <string>Acceso</string>
    </property>
    <addaction name="actionIniciar_Sesion"/>
    <addaction name="actionCerrar_Sesion"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAdministracion">
    <property name="title">
     <string>Administracion</string>
    </property>
    <addaction name="actionAgregar_Usuario"/>
    <addaction name="actionModificar_Usuario"/>
    <addaction name="actionEliminar_Usuario"/>
   </widget>
   <addaction name="menuAcceso"/>
   <addaction name="menuAdministracion"/>
   <addaction name="menuConfiguracion"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionIniciar_Sesion">
   <property name="icon">
    <iconset resource="../img/recursos.qrc">
     <normaloff>:/img/login.png</normaloff>:/img/login.png</iconset>
   </property>
   <property name="text">
    <string>Iniciar Sesion</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionCerrar_Sesion">
   <property name="icon">
    <iconset resource="../img/recursos.qrc">
     <normaloff>:/img/logout.png</normaloff>:/img/logout.png</iconset>
   </property>
   <property name="text">
    <string>Cerrar Sesion</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAlmacenamiento">
   <property name="icon">
    <iconset resource="../img/recursos.qrc">
     <normaloff>:/img/almacenamiento.png</normaloff>:/img/almacenamiento.png</iconset>
   </property>
   <property name="text">
    <string>Almacenamiento</string>
   </property>
  </action>
  <action name="actionComunicacion">
   <property name="icon">
    <iconset resource="../img/recursos.qrc">
     <normaloff>:/img/comunicacion.png</normaloff>:/img/comunicacion.png</iconset>
   </property>
   <property name="text">
    <string>Comunicacion</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="../img/recursos.qrc">
     <normaloff>:/img/quit.png</normaloff>:/img/quit.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAgregar_Usuario">
   <property name="icon">
    <iconset resource="../img/recursos.qrc">
     <normaloff>:/img/add.png</normaloff>:/img/add.png</iconset>
   </property>
   <property name="text">
    <string>Agregar Usuario</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionModificar_Usuario">
   <property name="icon">
    <iconset resource="../img/recursos.qrc">
     <normaloff>:/img/tools.png</normaloff>:/img/tools.png</iconset>
   </property>
   <property name="text">
    <string>Modificar Usuario</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionEliminar_Usuario">
   <property name="icon">
    <iconset resource="../img/recursos.qrc">
     <normaloff>:/img/remove.png</normaloff>:/img/remove.png</iconset>
   </property>
   <property name="text">
    <string>Eliminar Usuario</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionGetCantidad">
   <property name="text">
    <string>GetCantidad</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../img/recursos.qrc"/>
 </resources>
 <connections/>
</ui>
